src/asli/asli.py

```diff
@@ -197,10 +203,25 @@ def read_mask_data(self):
         """
         Reads in the Land-Sea mask file from <data_dir>/<mask_filename>
         """
+        # Check is the path is an s3 bucket
+        if self.data_dir.startswith("s3://"):
+            # Using utility function to set up s3 connection with the config file
+            # Passing s3 connection and specifying file bucket 
+            import s3fs
+
+            s3_lsm_bucket = s3fs.S3Map(
+                os.path.join(self.data_dir, self.mask_filename),
+                s3 = configure_s3_bucket(self.s3_config_filepath, self.s3_config_filename)
+            )
 
-        self.land_sea_mask = xr.open_dataset(
-            Path(self.data_dir, self.mask_filename)
-        ).lsm.squeeze()
+            # Using open_zarr to read in files, ie. we are expecting .zarr NOT .nc
+            self.land_sea_mask = xr.open_zarr(
+                s3_lsm_bucket, consolidated=True
+            ).lsm.squeeze()
```

Comment:
<nitpick>Slight repetition of `.lsm.squeeze()` (and similar for `.msl` below) now that we do it in the s3 and local file cases, but I'm inclined to fix it later since this package may eventually become more generic and we'd need to parameterize at that point.</nitpick>

Reply:
I would like to seen what nitpick tags get rendered as :grin: 

I agree re: the repetition (it's the same with `import s3fs` in the relevant blocks of code, which I hate but works haha)

Reply:
Boringly, they just disappear!